gradle/libs.versions.toml

```diff
@@ -51,12 +51,12 @@ flywayGradlePlugin = { module = "org.flywaydb:flyway-gradle-plugin", version.ref
 gax = { module = "com.google.api:gax", version = "2.5.0" }
 gcpCloudCore = { module = "com.google.cloud:google-cloud-core", version = "1.95.4" }
 gcpCloudStorage = { module = "com.google.cloud:google-cloud-storage", version = "1.117.1" }
-gcpDatastore = { module = "com.google.cloud:google-cloud-datastore", version = "1.107.1" }
+gcpDatastore = { module = "com.google.cloud:google-cloud-datastore", version = "2.2.10" }
```

Comment:
This is a big version jump, any migrations required?

Reply:
The API Safety check seemed to pass for this. My local setup was having issues running some of the DB connectivity tests but those occurred on main too.

I think for safety it would be great to just gradle build off someone else's machine to validate if these are env specific or if they are related to this change.